kevm-pyk/src/kevm_pyk/kproj/evm-semantics/evm.md

```diff
@@ -403,12 +403,12 @@ The `#next [_]` operator initiates execution by:
 -   `#changesState` is true if the given opcode will change `<network>` state given the arguments.
 
 ```k
-    syntax Bool ::= #changesState ( OpCode , WordStack ) [klabel(#changesState), function]
- // --------------------------------------------------------------------------------------
+    syntax Bool ::= #changesState ( OpCode , List ) [klabel(#changesState), function]
+ // ---------------------------------------------------------------------------------
 ```
 
 ```k
-    rule #changesState(CALL         , _ : _ : VALUE : _) => true  requires VALUE =/=Int 0
+    rule #changesState(CALL         , ListItem(_) ListItem(_) ListItem(VALUE) _) => true  requires VALUE =/=Int 0
```

Comment:
Why are we not using `ListItem(_)` or `.List` in the latest pattern match here instead of `_`? Sorry if this is a lack of knowledge about the context here.